INSTALL.md

```diff
@@ -206,7 +206,7 @@ make
 make install
 ```
 
-For details, see <https://trac.osgeo.org/grass/wiki/HowToGit>
+For details, see <https://github.com/OSGeo/grass/blob/main/doc/development/github_guide.md>
```

Comment:
There is already REQUIREMENTS.md linked as relative link, these are absolute, should we make it consistent?